### PR TITLE
Feature/nanoflow

### PR DESCRIPTION
**Pull request by McDoyen**
* Add nanoflow event

### CODE REVIEW

**Review by jseremba — COMMENTED** [review]
.

File: package.json [hunk 1]
```diff
@@ -58,7 +58,7 @@
     "karma-remap-istanbul": "^0.6.0",
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.3",
-    "mendix-client": "https://github.com/FlockOfBirds/Mendix-client-typing/archive/7.1.0.tar.gz",
+    "mendix-client": "https://github.com/FlockOfBirds/Mendix-client-typing/archive/7.13.1.tar.gz",
```
Comment (jseremba) [inline package.json]: Use the `npm` version
`mendix-client: 7.13.1`

**Review by jseremba — CHANGES_REQUESTED** [review]

File: src/RangeSlider.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+    <icon>iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAYAAACqaXHeAAAAAXNSR0IArs4c6QAAAoRJREFUeAHtmU9qwkAUxk02qdAeQPAGvYV00XYhpaAuKt0oPVSpm5IuVCilC+tCvIU3EHqAFqwb0/dsApk/iXkjFKJfNpM388375v0ykZhUKjhAAARAAARAAARAAARAAARAAARAAARAAASOi4DnWm4URd71+0+f5vfo9JzzeF60oGYwaZ48eRRw374H+3Q6nT61Pcq19aF2QfkHw+Fwbx8nALfTqLZar0NaVMNWIC1uXg2C7uul92kbL9rX7XZra/IhvdWH+ucB+YRh6OzjF11MouMrklc86xhMrHECHOfwdhTPsgZreE0cuBxiALzts658egGsiW+RdHfhc972JM668uk8jVib7it8LgZAmfleLHpItEpOAlh4rkSrmFAgBpD84OmJbLFEa5mf/OBZhowuiVaZLAZAd/ipkiE3kGiNRAKfikSrGDkAUOaXPgCA0l/CPQvADtgTYOmnG09Q9FBxtdlsHqmyuq26r7tnW3dm39nLfebYPw8sfd9/oP8PH2lf4xbIKz49sYTn9bg2ZekGAGX0CAIDAG8Tqnt5gLVvbwG9LuM3QBfo8eXbSvQ/f3pTFXuwZ6vVEvmMx2MnH2MH6AUfegwAh36Fd9WHHbCL0KGPO+wA77s4FInWyCrwqUi0ipEYQPzqW0mSFUi0lhz8ir3oIdEqOcUAaPZAyZAfSLRKJn7vr3TkBBKtnkb88EAvIOmDyHrGb331ZOmYvw1MmsEFtaIHmiQH+7Tb7RnFuT40Ph+NRs4+4h3ABfFHDy4wWaze8liscSqe87EPf/Sg00wfHmMNa3mOyyHeAYnJ304o/6expB60IAACIAACIAACIAACIAACIAACIAACIAACx0LgF7FjzreN3sQFAAAAAElFTkSuQmCC</icon>
```
Comment (jseremba) [inline src/RangeSlider.xml:5]: Invalid icon, Use
`iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAMAAACdt4HsAAAAMFBMVEUAAAD///9SuP9cXFxXV1dJsfdWVlZJsfhVVVVJsfdVVVVVVVVIsPhVVVVIsPdVVVXYHHm1AAAADnRSTlMAARkZSWVlvb3IyOPz8+LP6LYAAACWSURBVHgB7dXvCoMwDARwq/Z/m7z/2w6KhFZGBgtjfrjf14NTES7bUwEAOJ9by969SULpvQS3qfZIQ9zvyZF4SIf6/EiX6G5J4kvS3sGT8GsSWASlIJPIa1JYFKWgkWhr0ll0pYAma8KT3xSg4Kw80IQ/qqcUVP6mgKu9wP4JD/yNKDAPinnSzKNqnnXzYbGfNgCAf3oBWEEpQZdl/coAAAAASUVORK5CYII=`

**Review by jseremba — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (McDoyen) [reply package.json]: Updated to 7.13.2 instead

**Pull request merged**